.gitignore

```diff
@@ -148,3 +148,4 @@ imdb
 xsum
 coco128
 wmt_en_ro
+kinetics
```

Comment:
Is this from something else?

Reply:
It is the video dataset. I have them installed and in s3. Just preventing them to be added.